packages/cosmos/src/error.rs

```diff
@@ -645,10 +645,24 @@ impl Display for SingleNodeHealthReport {
             }) => write!(f, "Last error: {timestamp} ({age:?}): {error}")?,
         }
         if let Some(first_request) = self.first_request {
+            let since = (Utc::now() - first_request).num_minutes();
+
+            let rate_per_minute = {
+                let since = u64::try_from(since);
+                match since {
+                    Ok(since) => self.total_query_count.checked_div(since),
+                    Err(_) => None,
+                }
+            };
+            let rate_per_minute = match rate_per_minute {
+                Some(rpm) => rpm.to_string(),
+                None => "0verflow".to_owned(),
```

Comment:
I think there are two things that can lead to `None` here:
* Divide by zero as you pointed out
* Overflow from u64::try_from

Probably a better value would be "Computation issue", but not sure if it's helpful. Do you have any suggestions ? I can raise a new PR with a better description.

Reply:
Thanks, that is better. I have attempted it here but without `context` since we don't use anyhow anymore: https://github.com/Levana-Protocol/levana-cosmos-rs/compare/rps-err?expand=1